pbcommand/models/report.py

```diff
@@ -85,7 +85,7 @@ def __init__(self, id_):
             raise PbReportError("Type error. id '{i}' cannot be {t}.".format(i=id_, t=type(id_)))
 
         if not re.match('^[a-z0-9_]+$', id_):
-            msg = "id '{i}' for {x} must contain only alphanumeric or underscore characters".format(x=self.__class__.__name__, i=id_)
+            msg = "id '{i}' for {x} must contain only lower-case alphanumeric or underscore characters".format(x=self.__class__.__name__, i=id_)
```

Comment:
Human brains are poorly optimized for regex parsing, even coders.  (Well, mine is anyway.)